### PR TITLE
Feature/add categories

### PR DESCRIPTION
**Pull request by en-aleksandar-stojmenovic**

### CODE REVIEW

File: ProjectSetup/Contracts/V1/Requests/CreatePostRequest.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ProjectSetup.Contracts.V1.Requests
+{
+	public class CreatePostRequest
+	{
+		public Guid Id { get; set; }
```
Comment (bronjaharis) [inline ProjectSetup/Contracts/V1/Requests/CreatePostRequest.cs]: Hide this field from front end. Don't allow user to see the ID during category creation.

**Review by bronjaharis — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.vs/
```
Comment (zdravkor) [inline .gitignore:36]: Nema potrebe da pravis svoj gitignore. Visual Studio moze da ti generise standardni C# .gitignore

File: ProjectSetup/Contracts/V1/Requests/CreatePostRequest.cs (hunk 1 above)
Comment (en-aleksandar-stojmenovic) [reply ProjectSetup/Contracts/V1/Requests/CreatePostRequest.cs]: I removed it, thanks

File: .gitignore (hunk 2 above)
Comment (en-aleksandar-stojmenovic) [reply .gitignore:36]: Popravio sam, hvala

**Pull request merged**